requirements.txt

```diff
@@ -3,18 +3,21 @@ git+https://:@gitlab.cern.ch:8443/fts/fts-rest.git#egg=fts3
 git+https://github.com/DIRACGrid/pyGSI.git#egg=GSI
 
 # From pypi
+asn1
 certifi
 CMRESHandler>=1.0.0b4
 coverage>=4.0.3
 elasticsearch-dsl>=2.0.0
 funcsigs
+future
```

Comment:
Why is this needed?

Reply:
asn1 module requires it for builtins. I'm not sure why it haven't installed it as its dependency, but I clearly remember having to install it. Or was it that Travis didn't install it? It was quite log ago, I don't remember the details, but I'm sure it's because asn1 library (I uninstalled future on my test machine to verify and asn1 crashed on importing builtins).